Makefile

```diff
@@ -1,6 +1,8 @@
 include Makefile.env
 export DOCKER_TAGNAME ?= 0.0.0
 export KUBE_NAMESPACE ?= fybrik-system
+export OLD_CRD_VERSION ?= 0.7.0
```

Comment:
Consider using `LATEST_BACKWARD_SUPPORTED_VERSION` here as in the workflow above instead of `OLD_CRD_VERSION` to avoid future mismatch.

Reply:
I'm not sure that I undustand your point, where do you see future mismatch. I do want to have an option to deploy different versions of CRDs, Fybrik and the test module. (we can update Fybrik without updating the module version)

Reply:
P.S. there is difference in the Fybrik and AFM versions, the first is `0.7.0` and the second is `v0.7.0`